content/hardware/04.pro/boards/portenta-x8/tutorials/image-building/content.md

```diff
@@ -0,0 +1,211 @@
+---
+beta: true
+title: 'How To Build a Custom Image for Your Portenta X8'
+description: 'This tutorial teaches you how to compile a custom image for your Portenta X8 through USB'
+difficulty: intermediate
+tags:
+  - Embedded Linux
+  - Building
+  - Foundries.io
+author: 'Pablo Marquínez'
+hardware:
+  - hardware/04.pro/boards/portenta-x8
+---
+
+## Overview
+
+In this tutorial you will learn how to build an image for the Portenta X8 with the files provided from Foundries.io.
```

Comment:
```suggestion
In this tutorial you will learn how to build an image for the Portenta X8 with the files provided from Foundries.io.
Building your image locally is helpful to quickly debug certain aspects of the systems, like bootloader or kernel support.
Please keep in mind that images built locally cannot register to a FoundriesFactory and will not be OTA compatible, but this is a good alternative for those who do not have a FoundriesFactory subscription.
For FoundriesFactory subscribers, we strongly suggest to make use of the Factory continuous integration system for creating your images.
```

Reply:
then after this is merged we can work on another tutorial to help subscribers to learn how to trigger new builds